README.md

```diff
@@ -10,22 +10,19 @@
 
 ## Dependencies
 - **inotifywait** - used for watching for file changes with as little CPU usage as possible
+```sh
+sudo apt install inotify-tools
+```
 - **sunwait** - (optional) used for monitoring day/night cycle for night light feature
 - **geoclue2** - (optional) used for getting current location for day/night cycle for night light feature. Optional because locaiton can be set manually
 
-## How To Install
```

Comment:
Why remove the "How to Install" section? It is more generic than "How to Install - Ubuntu" and not everyone is using Ubuntu